src/routes/events.js

```diff
@@ -162,14 +162,14 @@ exports.plugin = {
             auxiliary: Joi.objectId(),
             sector: Joi.string(),
             address: Joi.object().keys({
-              street: Joi.string().allow(null, '').default(''),
-              zipCode: Joi.string().allow(null, '').default(''),
-              city: Joi.string().allow(null, '').default(''),
-              fullAddress: Joi.string().allow(null, '').default(''),
-              location: {
-                type: Joi.string().allow('', null),
-                coordinates: Joi.array().allow([], null),
-              },
+              street: Joi.string().required(),
+              zipCode: Joi.string().required(),
+              city: Joi.string().required(),
+              fullAddress: Joi.string().required(),
+              location: Joi.object().keys({
+                type: Joi.string().required(),
+                coordinates: Joi.array().length(2).required(),
+              }).required(),
```

Comment:
Il faut pas aussi mettre : `.when('type', { is: Joi.valid(INTERVENTION), then: Joi.required() }),
` ?

Reply:
pour la meme raison que au dessus, non il ne faut pas le mettre
- quand `address` est present, je veux tous ces champs
- address n'est pas obligatoire (meme pour une intervention > en cas de drag and drop on ne change que les date et auxiliaire / equipe)